package.json

```diff
@@ -351,6 +351,7 @@
     "@metamask/solana-wallet-snap": "^2.3.8",
     "@metamask/solana-wallet-standard": "^0.5.1",
     "@metamask/streams": "^0.4.0",
+    "@metamask/subscription-controller": "./subscription-controller.tgz",
```

Comment:
Would it be possible to use a preview build here instead?

Also, if you use a prepackaged file here, it would make sense to add it to the PR as well so that others can compile and test as well.